### DISCUSSION TITLE
ISO for 2.3.130

### DISCUSSION THREAD

**Discussion by Wilk4013** [Security-Onion-Solutions/securityonion#8512] [Unsupported Versions] [answered]
Could someone point me to the link to download the iso for 2.3.130? Having a problem with .140 and wnated to try and build with .130 and upgrade to .140 and I can'tseem to find the iso.
Thanks,
Wilk4013

**Comment by yomaoni** [comment, marked as answer]
https://download.securityonion.net/file/securityonion/securityonion-2.3.130-20220607.iso

**Reply by Wilk4013** [reply]
Thank you Yomaoni
